api/v1/loadtest_types.go

```diff
@@ -264,9 +264,9 @@ var FailedSettingDefaultsError = "FailedSettingDefaults"
 // in the Initializing state.
 var PodsMissing = "PodsMissing"
 
-// TimeoutExceeded is the reason string when the load test has not yet terminated
+// TimeoutErrored is the reason string when the load test has not yet terminated
 // but exceeded the timeout.
-var TimeoutExceeded = "TimeoutExceeded"
+var TimeoutErrored = "imeoutErrored"
```

Comment:
There is a typo here. I thought TimeoutExceeded was fine, but it is ok to change to TimeoutErrored as well.

Reply:
Right, I think it is a type of error reason, so it made scene to have the 'error' word in the reason itself.